.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -52,6 +54,21 @@ pipeline {
                 }
             }
         }
+        stage('Install build-chain tool') {
+            steps {
+                script {
+                    println "[INFO] Recovering build-chain version from composite action file"
```

Comment:
please sync the PR with main one about "Recovering" word too

Reply:
done, thanks!